controllers/upgrade_handlers.go

```diff
@@ -268,14 +271,80 @@ func (u *UpgHandler) autoRollbackIfEnabled(ibu *lcav1alpha1.ImageBasedUpgrade, m
 	return
 }
 
+// +kubebuilder:rbac:groups=sriovnetwork.openshift.io,resources=sriovnetworknodepolicies,verbs=get;list;watch;patch
+
+func (u *UpgHandler) sriovNetworkNodeStateWorkaround(ctx context.Context, msg string) error {
+	//
+	// Workaround: If the health checks fail, reporting that the SriovNetworkNodeState does not exist,
+	// add an annotation to the sriov default policy to kick the reconciler.
+	//
+	// After the postpivot, the various pods are all coming up at the same time, once kubelet is enabled by
+	// the postpivot service-unit. There is a bit of a race condition where the sriov-network-operator starts
+	// around the same time as the kube-apiserver. This means sriov-network-operator may start its reconcilers
+	// prior to being able to connect to etcd, which can be hidden by the use of cached API. When this happens,
+	// its reconcilers run with no CRs, but also no failures. It thinks the reconcilers were successful, and waits
+	// 5 minutes before running the reconcilers again.
+	//
+	// When this occurs, no SriovNetworkNodePolicies (other than default) or SriovNetworkNodeStates are created.
+	// This, in turn, results in a failure when trying to apply extra-manifests that include sriov config.
+	//
+	// Eventually, the periodic reconciler kicks in and creates the required CRs.
+	//
+	// The inclusion of the SriovNetworkNodeState health check blocks the Upgrade handler from attempting to apply
+	// the extra-manifests until it has a node in Succeeded state, protecting LCA from this failure. However, this
+	// LCA is waiting up to 5 minutes for the health checks to pass as a result, which increases the time required
+	// to perform an IBU.
+	//
+	// As a workaround, until such time that sriov-network-operator handles this scenario, this function kicks the
+	// reconciler by adding an annotation to the default SriovNetworkNodePolicy. This change to the CR triggers
+	// the reconciler to run, creating the necessary resources.
+	//
+	// If the health check failure message includes the message from the SriovNetworkNodeState health check that there
+	// were no SriovNetworkNodeState CRs, it is an indication that the sriov-network-operator reconciler ran without
+	// creating the CRs, which triggers the workaround to give the reconciler a little kick before requeuing. The health
+	// check should then be fine on the next pass of the IBU reconciler, as there will have been enough time for the CRs
+	// to have been created.
+	//
+
+	if !strings.Contains(msg, healthcheck.SriovNetworkNodeStateNotPresentMsg) {
+		return nil
+	}
+
+	policies := &sriovv1.SriovNetworkNodePolicyList{}
```

Comment:
You could consider leveraging  `Unstructured` instead of typed. Seems like too much import just for the def? (though I'm team typed)

https://github.com/kubernetes/client-go/tree/master/examples/dynamic-create-update-delete-deployment
https://github.com/openshift-kni/lifecycle-agent/blob/main/internal/extramanifest/extramanifest.go#L212-L215

Reply:
But we'd want the type for the health check. While we could likely use unstructured here in the workaround, we're accessing the status field. So we'd still be importing the same API. Unless you're suggesting there's a way we can drop typed queries for the health checks and still access the status fields?

Reply:
Yep exactly you could do something like `unstructured.NestedMap(u.UnstructuredContent(), "status")` to key out just the status or even the full with `u.UnstructuredContent()`...they both return map[string]interface{} so it's a bit awkward to use. 

Either way (typed vs Unstructured) it's good with me though so up to you

Reply:
Could be an interesting exercise, maybe we can reduce imports overall, as long as the resulting code doesn't end up ugly/unmaintainable